api/DocumentTimeline.json

```diff
@@ -12,7 +12,7 @@
             "version_added": "54"
           },
           "edge": {
-            "version_added": "≤79"
+            "version_added": "80"
```

Comment:
This is very surprising, can you confirm that in Edge 79 the `DocumentTimeline` constructor wasn't there, but `document.timeline` was? It just seems weird that this would have been disabled for just one release, when then change in Chrome happened way back in 54....

Reply:
Apparently I initially made a typo, and meant to put "84" as the version.  However, upon digging deeper into the data, I found that the API wasn't around in Chrome 54 either, but rather, it was introduced in Chrome 84.  It appears we need to correct __all__ of our Chrome data accordingly -- I've opened #9999 (nice!) to fix all of the Chromium data (and supersede this PR).